packages/core/src/infrastructure/database/migrations/2024-03-31T21:37:51Z-share-email.ts

```diff
@@ -28,19 +14,4 @@ export async function down(db: Kysely<any>) {
     .alterTable('students')
     .dropColumn('allow_email_share')
     .execute();
```

Comment:
I believe the down function can also be simplified to only this, because when we drop the column, this will remove the default and not null requirements. Please let me know if I'm missing something!

Reply:
You got it right! 🔥